File: file_write/ruby_file_write/file_write.rb
#!/usr/local/bin/ruby -w
#comments


def fileWrite( ifname, data )
  File.open(ifname, 'w') {
    |f|
    f.write(data)
  }
end

def main()
  if ARGV.length != 0 then
    puts "Usage: #{$0} <noargs>"
    exit(1)
  end
  #puts ARGV[0]
  fileWrite("output", "content")
end

main()

File: file_write/python_file_write/file_write.py
import sys
import os
import time

def fileWrite(ofname, data):
  ofp = open(ofname, "w")
  ofp.write(data)
  ofp.close()

fileWrite("output.txt", "uno does tres quatro cinco seis")

def main():
  if len(sys.argv) != 1:
    print ("Usage: %s <noargs>" % sys.argv[0])
    sys.exit()
  fileWrite(sys.argv[1], "data output")
  sys.exit(), end

main()